snapcraft.yaml

```diff
@@ -66,18 +82,40 @@ parts:
       - authz_core
       - unixd
       - alias
+      - ssl
+      - socache_shmcb
 
-    # Extra Apache configuration for Nextcloud (and PHP)
-    extra-configuration: src/nextcloud/apache_config
-
-    # Script to run before bringing up Apache
-    startup-script: src/nextcloud/setup_nextcloud
-
+    filesets:
+      exclude:
+        - -man
+        - -manual
+        - -htdocs
+        - -include
+        - -build
+        - -conf/httpd.conf
+        - -conf/magic
+        - -conf/original
+        - -conf/extra
+        - -bin/apachectl
+        - -bin/envvars*
     stage:
-      - -htdocs/.git*
+      - $exclude
     snap:
-      - -manual # No need to include the documentation in the .snap
-      - -htdocs/.git*
+      - $exclude
+
+  apache-customizations:
+    plugin: copy
+    files:
+      src/apache/scripts/*: bin/
+      src/apache/conf/*: conf/
+
+  nextcloud:
+    plugin: copy
+    source: https://github.com/kyrofa/nextcloud.git
+    source-tag: 9.0.50
```

Comment:
:disappointed: thanks, this feature obviously began before the upgrade, looks like it survived the rebase. Fixed!